instagram/src/components/CommentSection/CommentSectionContainer.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      comment: ''
+    };
+  }
+
+  componentDidMount() {
+   const id = this.props.postId;
+    if (localStorage.getItem(id)) {
+     this.setState({
+        comments: JSON.parse(localStorage.getItem(this.props.postId))
+      });
+    } else {
+     this.setComments();
+    }
+  }
+
+componentDidUpdate() {
+  localStorage.setItem(
```

Comment:
You have a convenience method to do this. You can simply call `this.setComments()` from here